sdk/core/Azure.Core/api/Azure.Core.netstandard2.0.cs

```diff
@@ -678,6 +678,14 @@ public abstract partial class TokenCredential
         public string[] Scopes { get { throw null; } }
         public string? TenantId { get { throw null; } }
     }
+    public partial class UserAgentValue
+    {
+        public UserAgentValue(System.Type packageType, string? applicationId = null) { }
+        public string? ApplicationId { get { throw null; } }
+        public System.Type PackageType { get { throw null; } }
+        public void Apply(Azure.Core.HttpMessage message) { }
```

Comment:
ApplyToMessage?

Reply:
My thinking here was that there is only one argument, and it is an HttpMessage. But I'm happy to change if there is consensus on this.

Reply:
Or SetOnMessage

Reply:
True, but users might think that you are applying something from the message to the UserAgent.

Reply:
I think `Apply` sounds sufficiently .NET-ey